src/util.rs

```diff
@@ -0,0 +1,12 @@
+/// Finds executable `bin_name` in PATH.
+pub fn find_executable(bin_name: &str) -> String {
```

Comment:
I'm surprised Rust has no good support for searching the path.

There is a crate we could use, which would be more portable:
https://crates.io/crates/which

Reply:
So was I! I'll use `which` then.